release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -326,6 +340,27 @@ You can now set the maximum length of the syslog message in the Ingress Controll
 
 For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-controller-configuration-parameters_configuring-ingress[Ingress Controller configuration parameters].
 
+[id="ocp-4-10-openstack-kuryr-resource-reduction"]
+==== Reduction of {rh-openstack} resources created by Kuryr
+
+For clusters that run on {rh-openstack}, Kuryr now only creates Neutron networks and subnets for namespaces that have at least one pod. Additionally, pools in a namespace are only populated after at least one pod is created in the namespace.
```

Comment:
nit: we need to specify the type of pods:
`for namespaces that have at least one pod on pods network.`
same applies to:
`after at least one pod on pods network is created in the namespace.`

Reply:
```suggestion
For clusters that run on {rh-openstack}, Kuryr now only creates Neutron networks and subnets for namespaces that have at least one pod on the pods network. Additionally, pools in a namespace are only populated after at least one pod on the pods network is created in the namespace.
```

@MaysaMacedo Like so? 

And definitely "pods" and not "pod's" or "pods'"?

Reply:
looks good to me